examples/logger.py

```diff
@@ -27,13 +27,13 @@ class LoggingHandler(logging.Handler):
     Sends a log message to connected function using emit.   
     """
 
-    def __init__(self, slotfunc: Callable[[str], None]):
+    def __init__(self, slotfunc: Callable[[str], Any]):
         """Extended.
 
         Connects the slotfunc to the signal.
 
         Args:
-            slotfunc: A slot function which is called when log record emitted.
+            slotfunc: A slot function which is called when a log record emitted.
```

Comment:
Since the logger emits the log records, so "a log record _is_ emitted.", i.e., you should use the passive form using "is".